pkg/agent/openflow/client.go

```diff
@@ -27,10 +27,22 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/agent/types"
 	"github.com/vmware-tanzu/antrea/pkg/agent/util"
 	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/util/runtime"
 	"github.com/vmware-tanzu/antrea/third_party/proxy"
 )
 
-const maxRetryForOFSwitch = 5
+const (
+	maxRetryForOFSwitch = 5
+
+	// We use OpenFlow Meter for packet-in rate limiting on OVS side.
+	// Meter Entry ID.
+	PacketInMeterIdNP = 1
+	PacketInMeterIdTF = 2
+	// Meter Entry Rate. It is represented as number of events per second. Over-rate
+	// packets will be dropped.
+	PacketInMeterRateNP = 100
+	PacketInMeterRateTF = 100
```

Comment:
you didn't like the idea of putting these in `pkg/agent/openflow/packetin.go`?

Reply:
LOL, since I don't have a strong preference and you said that's a "compromise" plan, then I go with your original suggestions.

Reply:
you brought up a good point that these were specific to PacketIn and I thought that it would make sense to move them to `pkg/agent/openflow/packetin.go` where we already have similar constants

Reply:
Thx. Done.